src/routers/project.js

```diff
@@ -0,0 +1,94 @@
+const express = require("express");
+const Project = require("../models/project");
+const auth = require("../middleware/auth");
+const AppError = require("../Errors/appError");
+const User = require("../models/user");
+
+const router = new express.Router();
+
+//Create Project
+router.post("/projects", auth, async (req, res, next) => {
+  const project = new Project({
+    ...req.body, //Express Spread Op
+    owner: req.user._id,
+  });
+  try {
+    await project.save();
+    res.status(201).send("New Project is added");
+  } catch (e) {
+    console.log(e);
+    return next(new AppError(e, 500));
```

Comment:
You can provide error stack in development mode but not in production Mode.
Just pass a message in production mode.